src/main.cpp

```diff
@@ -569,8 +573,63 @@ int main(int argc, char *argv[])
         }
     }
 
+    if (c == 'h') 
+    {
+         printf("Usage: wf-recorder [OPTION]... [FILE]...
+Screen recording of wlroots-based compositors
+
+With no FILE, start recording the current screen.
+
+  -a, --audio               Starts recording the screen with audio
```

Comment:
For the sake of completeness, the `--audio/-a` parameter takes an optional parameter, which is the name of the pulseaudio device to capture (the list can be obtained by running `pactl list sinks | grep Name`)